### ISSUE TITLE
🐛 Bug Report: unused code warning due to cached

### ISSUE THREAD

**Issue by Daniel-Valentine** [ferritreader/ferrit#26]
Clippy is warning about unused code:

```
warning: function `json_prime_cache` is never used
   --> src/client.rs:116:14
    |
116 | pub async fn json(path: String, quarantine: bool) -> Result<Value, String> {
    |              ^^^^
    |
    = note: `#[warn(dead_code)]` on by default
```

This appears to be an issue with the [cached](https://docs.rs/cached/latest/cached/) crate. I thought this was due to the dependency update that I did in a125b8a43e2cbffa7e41edfb907a4e2bbbbcd690, but checking out 57d304161b3af6bee4ff849f39d80e7a6d020510 (the last commit before the fork) and running `cargo check` yields the same warning.

**Comment by Daniel-Valentine** [comment]
[This is fixed in cached v0.37.0](https://github.com/jaemk/cached/blob/master/CHANGELOG.md#0370--cached_proc_macro0140). See jaemk/cached#121. An update to that dependency will resolve the issue.

```shell
$ cargo tree --depth=1 --prefix=none | grep cached
cached v0.37.0

$ cargo rustc -- --deny=warnings
   Compiling ferrit v0.1.0 (/home/daniel/Sources/ferrit)
    Finished dev [unoptimized + debuginfo] target(s) in 12.14s
```

**Comment by Daniel-Valentine** [comment]
Resolved in 13a8983.
